bot.py

```diff
@@ -0,0 +1,122 @@
+from config import *
+
+bot = telebot.TeleBot(TOKEN)
+
+
+@bot.message_handler(commands=['start'])
+def send_welcome(message):
+    if check_user(message.from_user.username):
+        text = f'Hello, {message.from_user.username}!'
+        bot.send_message(message.chat.id, text, reply_markup=menu_markup())
+    else:
+        text = f'Hello, {message.from_user.username}!\n\nAt first print your age, to filter news.'
+        bot.send_message(message.chat.id, text)
+        bot.register_next_step_handler(message, get_age)
+
+# Пользователь вводит возраст
+def get_age(message):
+    age = message.text
+    if age.isdigit():
+        sphere = []
+        text = "Perfect!\n\nNow let's choose sphere of news you want to know"
+        bot.send_message(message.chat.id, text, reply_markup=filter_markup())
+        bot.register_next_step_handler(message, creating_profile, age, sphere)
+    else:
+        bot.send_message(message.chat.id, 'Try to print your age, but not word!')
+        bot.register_next_step_handler(message, get_age)
+
+
+def creating_profile(message, age, sphere):
+    if message.text == 'Apple' or message.text == 'Tesla' or message.text == 'Top Business Headlines US' or message.text == 'Top Business Headlines TC' or message.text == 'WallStreet News':
+        if message.text in sphere:
+            sphere.remove(message.text)
+            bot.send_message(message.chat.id, f'Category *{message.text}* was deleted. You can choose more category or click *Create Profile*',
+                             reply_markup=filter_markup(), parse_mode="Markdown")
+            bot.register_next_step_handler(message, creating_profile, age, sphere)
+        else:
+            sphere.append(message.text)
+            bot.send_message(message.chat.id, f'Category *{message.text}* was added. You can choose more category or click *Create Profile*',
+                             reply_markup=filter_markup(), parse_mode="Markdown")
+            bot.register_next_step_handler(message, creating_profile, age, sphere)
+    elif message.text == 'Create Profile':
+        sphere_bit = ''
+        for SPHERE in SPHERES:
+            if SPHERE in sphere:
+                sphere_bit += '1'
+            else:
+                sphere_bit += '0'
+        
+        create_user(message.from_user.username, age, sphere_bit)
+
+        text = 'Your Profile has been successfully created!'
+        bot.send_message(message.chat.id, text, reply_markup=menu_markup())
+
+
+
+
+@bot.message_handler(content_types=["text"])
+def send_text(message):
+    if message.text == 'Show Actual News':
+        today = datetime.today()
+
+        TeslaRequest = requests.get(f'https://newsapi.org/v2/everything?q=tesla&from={str(today)[:10]}&sortBy=publishedAt&apiKey={API_TOKEN}').json()
```

Comment:
It's better to make lines shorter if possible. And maybe to create some separate template for the requests